frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -53,7 +58,11 @@ const Navbar = ({children}) => {
           />
           <nav className="flex flex-col gap-4">
             {navLinks.map((link, index) => (
-              <p key={index} className="text-lg" onClick={() => setShowMenu(false)}>
+              <p
+                key={index}
+                className="text-lg"
+                onClick={() => setShowMenu(false)}
+              >
                 <Link to={link.path}>{link.name}</Link>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)